apps/i18n/javalab/en_us.json

```diff
@@ -24,16 +25,19 @@
   "internalCompilerException": "Oops. We hit an error on our side while compiling your program. Try running it again. If this error continues, contact us at support@code.org. Be sure to include the connection ID {connectionId} in your message.",
   "internalException": "Oops. We hit an error on our side. Try running your program again. If this error continues, contact us at support@code.org. Be sure to include the connection ID {connectionId} in your message.",
   "internalRuntimeException": "Oops. We hit an error on our side while running your program. Try running it again. If this error continues, contact us at support@code.org. Be sure to include the connection ID {connectionId} in your message.",
+  "generatingResults": "Generating results...",
   "javaExtensionMissing": "Invalid file name. File names must end in '.java'.",
   "newFile": "New File",
   "noMainMethod": "Error: your program does not contain a main method.",
   "onlyVisibleToYou": "only visible to you",
+  "programCompleted": "Program completed.",
   "reOpen": "Re-open",
   "resolve": "Resolve",
+  "running": "Running...",
   "runtimeError": "Your code hit an exception while it was running.\n{cause}",
   "saveToBackpack": "Save to Backpack",
+  "startingPainter": "Starting painter.",
   "twoMainMethods": "Error: Your code can only have one main method. We found at least two classes with main methods.",
   "togglePeerReviewError": "Error enabling/disabling peer review, please try again.",
-  "backpackSaveInProgress": "Error: Save to Backpack is currently in progress, cannot start another Save to Backpack now.",
```

Comment:
Intentional?

Reply:
yeah, this should have been removed in a previous pr, we aren't using it anymore